<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QTabWidget" name="ViewsTabWidget">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ExpensesViewTab">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <attribute name="title">
        <string>Expenses View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Expenses View</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QTableView" name="accountView">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>60</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>New Line</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="0">
              <widget class="QLabel" name="priceperunitLabel">
               <property name="text">
                <string>â‚¬/unit</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="categoryLabel">
               <property name="text">
                <string>Category</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="4">
              <widget class="QLineEdit" name="labelEditField"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelLabel">
               <property name="text">
                <string>Label</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="priceperunitEditField"/>
             </item>
             <item row="1" column="3">
              <widget class="QLabel" name="qtyLabel">
               <property name="text">
                <string>Quantity</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>200</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QComboBox" name="categoryCB">
               <property name="editable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QSpinBox" name="qtySpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Add Line</string>
               </property>
               <property name="shortcut">
                <string>Return</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3" colspan="2">
              <widget class="QToolButton" name="toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDateEdit" name="dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Date</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AnalyticsTab">
       <attribute name="title">
        <string>Analytics</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTabWidget" name="ChartsViewsTabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Pies</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="2" column="0">
             <widget class="QDateEdit" name="yearEdit">
              <property name="currentSection">
               <enum>QDateTimeEdit::YearSection</enum>
              </property>
              <property name="displayFormat">
               <string>yyyy</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDateEdit" name="monthEdit">
              <property name="currentSection">
               <enum>QDateTimeEdit::MonthSection</enum>
              </property>
              <property name="displayFormat">
               <string>MM/yyyy</string>
              </property>
              <property name="calendarPopup">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QChartView" name="graphicsView"/>
            </item>
            <item row="1" column="1">
             <widget class="QChartView" name="graphicsView_2"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="yearTotalLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="monthTotalLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Bars</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="1">
             <widget class="QChartView" name="graphicsView_3"/>
            </item>
            <item row="1" column="2">
             <widget class="QListView" name="monthsListView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::MultiSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="containerLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuBudget">
     <property name="title">
      <string>Budget</string>
     </property>
     <addaction name="actionTable_View"/>
     <addaction name="actionExpenses_View"/>
    </widget>
    <addaction name="menuBudget"/>
    <addaction name="separator"/>
    <addaction name="actionAnalytics"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAnalytics">
   <property name="text">
    <string>Analytics</string>
   </property>
  </action>
  <action name="actionTable_View">
   <property name="text">
    <string>Budget Planning</string>
   </property>
  </action>
  <action name="actionExpenses_View">
   <property name="text">
    <string>Expenses View</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getNewCategoryFromFormSLOT()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addNewLineSLOT()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>680</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ViewsTabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>tabChangedSLOT()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>monthEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>monthlyAnalyticsDateChangedSLOT()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yearEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>yearlyAnalyticsDateChangedSLOT()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChartsViewsTabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>chartsViewTabChangedSLOT()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>monthsListView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>monthViewItemClickedSLOT()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getNewCategoryFromFormSLOT()</slot>
  <slot>addNewLineSLOT()</slot>
  <slot>tabChangedSLOT()</slot>
  <slot>monthlyAnalyticsDateChangedSLOT()</slot>
  <slot>yearlyAnalyticsDateChangedSLOT()</slot>
  <slot>chartsViewTabChangedSLOT()</slot>
  <slot>monthViewItemClickedSLOT()</slot>
 </slots>
</ui>
